modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleList/RuleList.es.js

```diff
@@ -44,6 +45,12 @@ class RuleList extends Component {
 				true
 			)
 		);
+
+		var tooltip = ClayTooltip.init();
```

Comment:
@alinedoleron, why are you using `var` instead of `let` or `const`?

Reply:
I didn't noticed. It was a copy and paste from a piece of code of Clay's demo.